stagecraft/apps/collectors/tasks.py

```diff
@@ -17,6 +17,14 @@ def log(message):
 
 
 @shared_task
+def run_collectors_by_type(*args):
+    for slug in args:
+        collector_type = CollectorType.objects.get(slug=slug)
+        slugs = [c.slug for c in collector_type.collector_set.all()]
```

Comment:
Can we check this isnt running a query for every single collector and do the eager loading stuff if it is?